homeassistant/components/syncthru/sensor.py

```diff
@@ -3,98 +3,110 @@
 import logging
 import voluptuous as vol
 
-from homeassistant.const import (
-    CONF_RESOURCE, CONF_HOST, CONF_NAME, CONF_MONITORED_CONDITIONS)
+from homeassistant.const import (CONF_RESOURCE, CONF_HOST, CONF_NAME,
```

Comment:
Please revert all formatting changes. They can come in a separate PR.

Reply:
Should be done